pkgs/tools/system/mlc/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, patchelf, lib }:
+stdenv.mkDerivation rec {
+  pname = "mlc";
+  version = "3.9";
+
+  src = fetchurl {
+    url = "https://software.intel.com/content/dam/develop/external/us/en/protected/mlc_v${version}.tgz";
+    sha256 = "1x7abm9hbv9hkqa3cgxz6l04m3ycyl40i4zgx1w819pc10n6dhdb";
+  };
+
+  sourceRoot = "Linux";
+
+  nativeBuildInputs = [ patchelf ];
+
+  phases = [ "unpackPhase" "installPhase" "patchPhase" ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp mlc $out/bin/mlc
+    chmod +x $out/bin/mlc
```

Comment:
```suggestion
    install -Dm755 mlc $out/bin/
```

Reply:
Done. I also removed the `mkdir $out/bin` since that's handled by `install` as well.